### ISSUE TITLE
尝试UIE finetune时不稳定coredump

### ISSUE THREAD

**Issue by hnlslyp** [PaddlePaddle/Paddle#46144]
### 问题描述 Please describe your issue

/ssd1/liuyaping/env/lib/python3.7/site-packages/pandas/util/testing.py:20: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  from pandas._libs import testing as _testing
/ssd1/liuyaping/env/lib/python3.7/site-packages/scipy/sparse/sputils.py:16: DeprecationWarning: `np.typeDict` is a deprecated alias for `np.sctypeDict`.
  supported_dtypes = [np.typeDict[x] for x in supported_dtypes]
/ssd1/liuyaping/env/lib/python3.7/site-packages/scipy/special/orthogonal.py:81: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  from numpy import (exp, inf, pi, sqrt, floor, sin, cos, around, int,
/ssd1/liuyaping/env/lib/python3.7/site-packages/scipy/linalg/__init__.py:217: DeprecationWarning: The module numpy.dual is deprecated.  Instead of using dual, use the functions directly from numpy or scipy.
  from numpy.dual import register_func
[2022-09-16 18:28:44,295] [    INFO] - Downloading resource files...
[2022-09-16 18:28:44,297] [    INFO] - We are using <class 'paddlenlp.transformers.ernie.tokenizer.ErnieTokenizer'> to load 'uie-base'.
W0916 18:28:44.334818 64351 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 10.2, Runtime API Version: 10.2
W0916 18:28:44.337807 64351 gpu_resources.cc:91] device: 0, cuDNN Version: 7.0.
train.sh: line 19: 64351 Segmentation fault      /ssd1/liuyaping/env/bin/python3.7 finetune.py --train_path /ssd1/liuyaping/practic/pugc/train.txt --dev_path /ssd1/liuyaping/practic/pugc/dev.txt --save_dir ./checkpoint --learning_rate 1e-5 --batch_size 16 --max_seq_len 256 --num_epochs 300 --model uie-base --seed 1000 --logging_steps 10 --valid_steps 100 --device gpu

**Comment by wjj19950828** [comment]
@hnlslyp  看起来是PaddleNLP的使用问题，建议去[PaddleNLP](https://github.com/PaddlePaddle/PaddleNLP) Issue下提问～
